### ISSUE TITLE
Formatting of "minishift start" output is broken for generic driver

### ISSUE THREAD

**Issue by LalatenduMohanty** [minishift/minishift#2501]
Here is the broken formatting

```
-- Starting local OpenShift cluster using 'generic' hypervisor ...
...... OK
```

Complete output :

```
$ minishift start --vm-driver generic --remote-ipaddress xxx.xxx.xxx.xxx --remote-ssh-user root --remote-ssh-key ~/.ssh/id_rsa
-- Starting profile 'minishift'
-- Checking if provided oc flags are supported ... OK
-- Starting local OpenShift cluster using 'generic' hypervisor ...
...... OK
-- OpenShift cluster will be configured with ...
   Version: v3.9.0
-- Copying oc binary from the OpenShift container image to VM .. OK
-- Starting OpenShift cluster ...................
Using nsenter mounter for OpenShift volumes
Using public hostname IP 139.59.22.211 as the host IP
Using 139.59.22.211 as the server IP
Starting OpenShift using openshift/origin:v3.9.0 ...
OpenShift server started.

The server is accessible via web console at:
    https://139.59.22.211:8443

You are logged in as:
    User:     developer
    Password: <any value>

To login as administrator:
    oc login -u system:admin
```

**Comment by gbraad** [comment]
```
-- Starting local OpenShift cluster using 'generic' hypervisor ...
...... OK
```

is this also about a possible newline?

**Comment by LalatenduMohanty** [comment]
Resolved via #2503
